common.blocks/select/select.en.md

```diff
@@ -209,13 +213,13 @@ The modifier provides a focus to the block.
 
 #### `theme` modifier
 
-Available value: `'islands'`.
+Acceptable value: `'islands'`.
 
 Use case: `BEMJSON`.
 
-The modifier provides a custom design to the block.
+The modifier gives the block a custom design.
```

Comment:
The sentence is correct. The verb **give** (without to) has more suitable meaning.